src/standalone/index.js

```diff
@@ -58,10 +58,12 @@ function autocomplete(selector, options, datasets, typeaheadObject) {
   _.each(['open', 'close', 'getVal', 'setVal', 'destroy'], function(method) {
     inputs.autocomplete[method] = function() {
       var methodArguments = arguments;
+      var result;
       inputs.each(function(j, input) {
         var typeahead = zepto(input).data(typeaheadKey);
-        typeahead[method].apply(typeahead, methodArguments);
+        result = typeahead[method].apply(typeahead, methodArguments);
       });
+      return result;
```

Comment:
But it should already work because of https://github.com/algolia/autocomplete.js/blob/master/src/standalone/index.js#L56-L69